### PR TITLE
Fixed a few typos.

### PR DESCRIPTION
**Pull request by paulmcfe**
### Description

Fixed a few typos.

### Motivation

These docs should be grammatically as well as factually correct.

### CODE REVIEW

File: files/en-us/web/css/transition-duration/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The **`transition-duration`** [CSS](/en-US/docs/Web/CSS) property sets the lengt
 
 {{EmbedInteractiveExample("pages/css/transition-duration.html")}}
 
-You may specify multiple durations; each duration will be applied to the corresponding property as specified by the {{ cssxref("transition-property") }} property, which acts as a master list. If there are fewer durations specified than in the master list, the user agent repeat the list of durations. If there are more durations, the list is truncated to the right size. In both case the CSS declaration stays valid.
+You may specify multiple durations; each duration will be applied to the corresponding property as specified by the {{ cssxref("transition-property") }} property, which acts as a master list. If there are fewer durations specified than in the master list, the user agent repeats the list of durations. If there are more durations, the list is truncated to the right size. In both cases, the CSS declaration stays valid.
```
Comment (dipikabh) [inline files/en-us/web/css/transition-duration/index.md:14]:
```suggestion
You may specify multiple durations; each duration will be applied to the corresponding property as specified by the {{ cssxref("transition-property") }} property, which acts as a master list. If the number of specified durations is less than in the master list, the user agent repeats the list of durations. If the number of specified durations is more than in the master list, the list is truncated to the right size. In both the cases, the CSS declaration stays valid.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hello @paulmcfe, welcome to the MDN project!

Your changes look good. I have suggested a few more edits.

File: files/en-us/web/css/transition-duration/index.md (hunk 1 above)
Comment (paulmcfe) [reply files/en-us/web/css/transition-duration/index.md:14]: These edits are great: they make the text easier to read and understand. I would only say that "In both the cases" sounds off to me. I'd leave it as "In both cases".

Comment (dipikabh) [reply files/en-us/web/css/transition-duration/index.md:14]: Great, you can then push a new commit with these changes (minus the last "the").

**Comment by dipikabh** [comment]
Thanks for your patience, @paulmcfe. If you can just commit the edits, I can approve and merge the pull request.

**Review by dipikabh — APPROVED** [review]
Thanks 👍

**Pull request merged**